hazelcast-jet-s3/src/test/java/com/hazelcast/jet/s3/SinkTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.s3;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.DeleteObjectsRequest;
+import com.amazonaws.services.s3.model.ObjectListing;
+import com.amazonaws.services.s3.model.S3Object;
+import com.amazonaws.services.s3.model.S3ObjectSummary;
+import com.hazelcast.jet.IMapJet;
+import com.hazelcast.jet.JetInstance;
+import com.hazelcast.jet.core.JetTestSupport;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.Sources;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.List;
+import java.util.Map;
+
+import static com.amazonaws.regions.Regions.US_EAST_1;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class SinkTest extends JetTestSupport {
+
+    private static String bucketName = "jet-s3-connector-test-bucket-sink";
+    private static String accessKeyId;
+    private static String accessKeySecret;
+    private static AmazonS3 client;
+
+    @BeforeClass
+    public static void setup() {
+        accessKeyId = getSystemPropertyOrEnv("S3_ACCESS_KEY_ID");
+        accessKeySecret = getSystemPropertyOrEnv("S3_ACCESS_KEY_SECRET");
+        client = S3Utils.client(accessKeyId, accessKeySecret, US_EAST_1);
+    }
+
+    @Before
+    public void deleteObjects() {
```

Comment:
Does this leaves any files on the S3 bucket when a test fails for example ? So I think it is better to add also an `@After` hook to cleanup the bucket.